test/services/katello/pulp3/repository_list_test.rb

```diff
@@ -23,16 +23,14 @@ def test_list_with_pagination
           User.current = users(:admin)
           @primary = FactoryBot.create(:smart_proxy, :default_smart_proxy, :with_pulp3)
 
-          repo1 = katello_repositories(:pulp3_file_1)
+          repo1 = katello_repositories(:generic_file)
```

Comment:
This test wasn't really written properly, notice its creating the same repo in pulp3 twice.